lib/puppetserver/ca/action/list.rb

```diff
@@ -47,6 +47,9 @@ def self.parser(parsed = {})
             opts.on('--all', 'List all certificates') do |a|
               parsed['all'] = true
             end
+            opts.on('--format FORMAT', Array, 'Output in parsable json format') do |f|
```

Comment:
This type shouldn't be an Array, IMO. That would be for a description like `--format FORMAT[,FORMAT]` (see `--certname` below) or usage like `--format json,text` which... I don't know how or why you'd want it output as multiple formats at once. And it'd be misleading to allow that since we then just take the first format passed.

Reply:
I also think this help should say, "Valid formats are: text (default), json" or similar.

And then I think we should handle treating this key as an "enum" of either "text" or "json" (maybe symbols??) What do others think of that??

Reply:
I see what you meant.  I was thinking of something similar to how I would be accessing ARGV so I did that.